manage

```diff
@@ -29,6 +29,11 @@ pylint.FILES() {
     grep -l -r --include \*.py '^#[[:blank:]]*lint:[[:blank:]]*pylint' searx searx_extra tests
 }
 
+YAMLLINT_FILES=()
+while IFS= read -r line; do
+   YAMLLINT_FILES+=("$line")
```

Comment:
It should ignore files defined by `.gitignore`.
More into details: `make themes` will install different NPM packages, some of them includes .yml files which don't follow the yamllint format.

Reply:
fixed to: `git ls-files './tests/*.yml' './searx/*.yml'` / see [9ce7dc6f](https://github.com/searxng/searxng/pull/105/commits/9ce7dc6f2d4b5dd14b0f0ff04ff9f7a333595ef9#diff-181229424893bb65d94a74c2132b8b9e5adfe851464fdb5cb9f49e8a8204be7bR35)